sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/google-oauth2.yaml

```diff
@@ -55,16 +55,37 @@ provider:
         name: Google OAuth client secrets should not be disclosed
       detection:
         matching:
-          # This regex looks for ".apps.googleusercontent.com", followed by "secret", followed by the target secret.
-          # - There can be up to 5 lines of text between ".apps.googleusercontent.com" and "secret", as long as they
-          #   don't contain "}" or ")".
+          # This regex does the following:
+          #
+          # 1. Looks for ".apps.googleusercontent.com"
+          #   - There can be up to 5 lines of text between ".apps.googleusercontent.com"
+          #     and "secret", as long as they don't contain "}" or ")".
           #   - 5 lines has been chosen based on data in SourceGraph search results.
+          #   - 600 corresponds to 120 * 5 (120 generally being the line limit)
           #   - The presence of end braces likely means that any following text is unrelated.
-          # - The target secret can be on the same line as "secret" or the following line. It must appear within 50
-          #   characters of "secret" (same line) or the start of the line (following line).
-          #   - 50 characters is an arbitrary distance and should allow for whitespace and a prefix of some sort. The
-          #     further that the target secret is from the keyword "secret", the less likely it is to be related.
-          pattern: "(?i)(?:\\.apps\\.googleusercontent\\.com\\b(?:[^})\\r\\n]*\\r?\\n){0,5}?[^})\\r\\n]*)(?:secret(?:[^})\\r\\n]*\\r?\\n)?[^})\\r\\n]{0,50})\\b([A-Za-z0-9_\\-]{24})\\b"
+          #
+          # 2. Searches for "secret"
+          #  - The target secret can be on the same line as "secret" or the following line.
+          #  - It must appear within 50 characters of "secret" (same line) or
+          #    the start of the line (following line).
+          #   - 50 characters is an arbitrary distance and should allow for
+          #   whitespace and a prefix of some sort.
+          #
+          #   The further that the target secret is from the keyword "secret",
+          #   the less likely it is to be related.
+          #
+          # 3. Then looks for the target secret.
+          pattern: "(?i)\
+                    (?:\
+                        \\.apps\\.googleusercontent\\.com\\b\
+                        (?:[^})\\r\\n]*+\\r?\\n){0,5}\
+                        [^})\\r\\n]{0,600}\
```

Comment:
The 600 (5 * 120) does not really make sense here because you are not matching any line breaks.
So basically you are matching **one** line with up to 600 characters. I don't think this is your intention.

Reply:
Line breaks are represented by `\n` so I don't get what you mean: it's actually matching line breaks